src/download.ts

```diff
@@ -133,6 +128,9 @@ export async function getSourceFiles(filePaths: Map<number, string>): Promise<Ma
 			}
 			currentFileStrings!.set(sourceString.identifier, sourceString.text as string);
 		}
+		if (projectStrings.length !== 500) {
+			break;
+		}
```

Comment:
What's the meaning behind this special number `500`?

Reply:
The max amount of items (in this case strings) you can retrieve from the API is 500. What I do here is check if I have reached that limit in my request and if yes send another request with an offset of 500.

Reply:
Ah, that makes sense. Could you set a variable `MAX_API_PAGESIZE` elsewhere and use it here instead to make it more readable?